src/test/acceptance/features/are-you-pregnant.feature

```diff
@@ -0,0 +1,26 @@
+Feature:
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to select whether or not I am pregnant
+
+  Background:
+    Given I am on the are you pregnant page
+
+  Scenario: No option is select
+    Then No option is selected
+
+  Scenario: Yes and No are displayed
+    Then Yes and No options are displayed
+
+  Scenario Outline: Select that an option
```

Comment:
"Select that an option" -> "Select an option"
Worth bearing in mind that in the next story we'll be changing what happens when selecting 'yes', so this will change anyway at that point. We can probably leave this for now.